qcfractal/server.py

```diff
@@ -100,9 +100,45 @@ def __init__(
             logfile_prefix: str=None,
 
             # Queue options
-            queue_socket: 'queue_adapter'=None,
+            queue_socket: 'BaseAdapter'=None,
             max_active_services: int=20,
             heartbeat_frequency: int=300):
+        """QCFractal initialization
+
+        Parameters
+        ----------
+        name : str, optional
+            The name of the server itself, provided when users query /information
+        port : int, optional
+            The port the server will listen on.
+        loop : IOLoop, optional
+            Provide an IOLoop to use for the server
+        compress_response : bool, optional
+            Automatic compression of responses, turn on unless behind a proxy that
+            provides this capability.
+        security : Optional[str], optional
+            The security options for the server {None, "local"}. The local security
+            option uses the database to cache users.
+        allow_read : bool, optional
+            Allow unregistered to perform GET operations on Molecule/KeywordSets/KVStore/Results/Procedures
+        ssl_options : Optional[Dict[str, str]], optional
+            None, automatically creates self-signed SSL certificates. False, turns off SSL entirely. A user can also supply a dictionary of valid certificates.
+        storage_uri : str, optional
+            The database URI that the underlying storage socket will connect to.
+        storage_project_name : str, optional
+            The project name to use on the database.
+        query_limit : int, optional
+            The maximum number of entries a query will return.
+        logfile_prefix : str, optional
+            The logfile to use for logging.
+        queue_socket : BaseAdapter, optional
+            An optional Adapter to provide for server to have limited local compute.
+            Should only be used for testing and interactive sessions.
+        max_active_services : int, optional
+            The maximum number of active Services that can be running at any given time.
+        heartbeat_frequency : int, optional
```

Comment:
Does this strictly have to be `int`? There are some tests in `test_manager` which pass a float

Reply:
No, but close enough.